sql/catalyst/src/test/scala/org/apache/spark/sql/types/DataTypeSuite.scala

```diff
@@ -584,6 +585,69 @@ class DataTypeSuite extends SparkFunSuite {
     true,
     ignoreNullability = true)
 
+  def checkEqualsStructurallyByName(
+      from: DataType,
+      to: DataType,
+      expected: Boolean,
+      caseSensitive: Boolean = false): Unit = {
+    val testName = s"equalsStructurallyByName: (from: $from, to: $to, " +
```

Comment:
Maybe we should add a JIRA prefix.

Reply:
sure, wasn't sure if that was primarily for bug or when to not add it